witness/src/Dscp/Witness/Web/Logic.hs

```diff
@@ -58,9 +53,8 @@ getAccountState addr = do
         }
 
 -- | Submit transaction.
-submitUserTx
-    :: (MonadIO m, MonadThrow m, MonadReader ctx m, HasLens' ctx MempoolVar)
-    => TxWitnessed -> m ()
+submitUserTx :: WitnessWorkMode ctx m => TxWitnessed -> m ()
 submitUserTx tw = do
     mempool <- view (lensOf @MempoolVar)
     addTxToMempool mempool (GMoneyTxWitnessed tw)
+    relayTx (GMoneyTxWitnessed tw)
```

Comment:
I suppose now you can remove line above since transaction will occur in mempool anyway?

Reply:
Hmm... yes, by the way.